openfisca_france/model/mesures.py

```diff
@@ -254,6 +251,85 @@ def formula(foyer_fiscal, period):
         return csg_cap_lib + prelsoc_cap_lib + crds_cap_lib
 
 
+class impots_directs_menage(Variable):
```

Comment:
@benjello 

Toutes les variables qui suivent, ajoutées dans `taxipp`, ne comportent pas de description.
Par ailleurs, `impots_directs_menage ` est un nom étrange, vu que `impots_directs` est déjà une variable qui concerne les ménages.

Est-ce que ces variables ont vocation à être dans le modèle commun de la législation ? Si oui, est-ce que tu peux me donner pour chacune d'entre elles une description et un nom de variable plus explicite ?

Merci !

Reply:
@fpagnoux : on utilise les variables terminant par `_menage` pour sortir des stats sur la décomposition des niveaux de vie par décile (le niveau de vie étant par définition au niveau du ménage). La variable `revenu_disponible_famille` est aussi pour nos besoins spécifiques. Donc, toutes ces variables n'ont pas vocation à entrer dans OF France. Si tu peux les ajouter dans la réforme `taxipp`, comme tu as fait pour le reste, ça serait top. Pour la signification, les variables finissant par `_menage` sont des montant au niveau du ménage ramené au nombre d'unité de consommation (du coup, la différence entre `impots_directs` et `impots_directs_menage`, c'est la division du nombre d'UC). Ce sont de très mauvais noms en effet... Ne T'occupe pas des docstrings, on le fera. Car je pense qu'il faut surtout changer le nom de ces variables, ce qui implique de faire ces changements là où elles sont utilisées.

Reply:
Ok, super, je déplace tout ça sur la réforme taxipp !